README.md

```diff
@@ -0,0 +1,28 @@
+# Paketo Noble Base Stack
+
+## What is a stack?
+See Paketo's [stacks documentation](https://paketo.io/docs/concepts/stacks/).
+
+## What is this stack for?
+Ideal for:
+- Java apps and .NET Core apps
+- Go apps that require some C libraries
+- Node.js/Python/Ruby/etc. apps **without** many native extensions
+
+## What's in the build and run images of this stack?
+This stack's build and run images are based on Ubuntu Noble Numbat.
+
+- To see the **list of all packages installed** in the build or run image for a given release,
+see the `noble-base-stack-{version}-build-receipt.cyclonedx.json` and 
+`noble-base-stack-{version}-run-receipt.cyclonedx.json` attached to each
+[release](https://github.com/paketo-buildpacks/noble-base-stack/releases). For a quick overview
+of the packages you can expect to find, see the [stack descriptor file](stack/stack.toml).
+
+- To generate a package receipt based on existing `build.oci` and `run.oci` archives, use [`scripts/receipts.sh`](scripts/receipts.sh).
+
+## How can I contribute?
+Contribute changes to this stack via a Pull Request. Depending on the proposed changes,
+you may need to [submit an RFC](https://github.com/paketo-buildpacks/rfcs) first.
+
```

Comment:
Was the removal of this line from the Jammy equivalent on purpose versus updating it to reference noble?

To generate a package receipt based on existing build.oci and run.oci archives, use [scripts/receipts.sh](https://github.com/paketo-buildpacks/jammy-base-stack/blob/e0ff3a8e1b4f767c7803d8c8f8d5162afe446116/scripts/receipts.sh).

Reply:
never mind, read it wrong